source/transfer_list.rst

```diff
@@ -192,7 +192,7 @@ may be represented in a different kind of well-established handoff data
 structure (e.g. FDT [DT]_, HOB [PI]_) that is inserted into the TL as a single
 TE. The same tag ID may occur multiple times in the TL to represent multiple
 instances of the same kind of object. Tag layouts (including the meaning of all
-fields) are considered stable after being added to this specification and may
+fields) are considered stable after being added to this specification and must
```

Comment:
Start of this paragraph should say "Tags should have a simple ...".

Reply:
Done